src/cusum.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RECYCLE_SRC_CUSUM_H_
+#define RECYCLE_SRC_CUSUM_H_
+
+#include "pyre.h"
+#include "cyclus.h"
+
+namespace recycle {
+
+class Cusum {
+
+private:
+double oldCusum = 0;
+double cusum_val;
+double mean;
+double newMean;
+double newVal;
+double variance;
+double magnitude;
+double threshold;
+
+int startup;
+int count;
+
+bool divert;
+
+public:
```

Comment:
google style guide for c++ (the one we follow in the cyclus community) suggest:
`A class definition should usually start with a public: section, followed by protected:, then private:. Omit sections that would be empty.`